src/Serialization/Address.js

```diff
@@ -1,146 +1,112 @@
-var CardanoWasm;
-if (typeof BROWSER_RUNTIME != 'undefined' && BROWSER_RUNTIME) {
-    CardanoWasm = require('@ngua/cardano-serialization-lib-browser');
-} else {
-    CardanoWasm = require('@ngua/cardano-serialization-lib-nodejs');
-}
+const CardanoWasm = require("@ngua/cardano-serialization-lib-nodejs");
```

Comment:
This breaks the browser integration, so this change needs to be reverted

Reply:
Strange, I don't remember dealing with this as a conflict during merges. This branch is definitely too long-lived